ui/components/FlipCard/FlipCard.style.js

```diff
@@ -0,0 +1,31 @@
+import { styled } from "@mui/material/styles";
+
+export const Container=styled('div')(({theme})=>({
+  position:"relative",
+  height:"100%",
+  borderRadius : theme.spacing(1),
+}))
+
+export const InnerCard=styled('div')(({flipped,theme})=>({
+      padding : theme.spacing(2),
+      borderRadius : theme.spacing(1),
+      position:"relative",
+      height:"100%",
+      transformStyle:"preserve-3d",
+      transition:"all 0.5s ease",
+      boxShadow : "0 4px 8px 0 rgba(0,0,0,0.2)",
+      backgroundColor : "#fff",
+      cursor : "pointer",
+      transform:`${flipped && 'rotateY(180deg)'}`
+}))
+
+export const FrontCard=styled('div')(()=>({
+  height:"100%",
```

Comment:
@subhradwip01  Remove the height , as this is causing bug in the back side  of the card
![Screenshot (189)](https://user-images.githubusercontent.com/75125203/180500115-df8588a2-1c08-49b7-8309-0b9a383a79f9.png)